### PR TITLE
Accessibility retest #2

### PR DESCRIPTION
**Pull request by jonnystening**

### CODE REVIEW

File: components/breadcrumb/examples/00-default.js [hunk 1]
```diff
@@ -10,16 +10,25 @@ function Example({ brand }) {
...
+			<h3>As links</h3>
 			<Breadcrumb>
 				<Crumb href="#/" text="Home" />
 				<Crumb href="#/personal-banking/" text="Personal" />
 				<Crumb href="#/credit-cards/" text="Credit cards" />
 			</Breadcrumb>
 
-			<hr />
-
-			<h2>Declarative as router</h2>
```
Comment (dominikwilkowski) [inline components/breadcrumb/examples/00-default.js:22]: It would be good to see the `tag` prop in the examples somewhere

Comment (jonnystening) [reply components/breadcrumb/examples/00-default.js:22]: yes, could I get either you @dominikwilkowski or you @jaortiz to assist?

File: components/breadcrumb/examples/00-default.js [hunk 2]
```diff
@@ -5,6 +5,9 @@ import { Breadcrumb, Crumb } from '@westpac/breadcrumb';
 
 import { Intopia } from '../../../helpers/example/components/Intopia.js';
 
+// Crumb tag as a component example
+const CrumbTag = props => <a {...props} onClick={() => console.log(`Clicked crumb link`)} />;
```
Comment (jonnystening) [inline components/breadcrumb/examples/00-default.js:9]: Added an example showing `tag` prop taking a component...

File: components/breadcrumb/src/Crumb.js [hunk 3]
```diff
@@ -62,22 +69,25 @@ export const Crumb = ({ current, href, text, onClick, overrides, ...rest }) => {
...
 
 	/**
 	 * The text of the crumb
 	 */
 	text: PropTypes.string.isRequired,
 
+	/**
+	 * Component tag, may receive a react-router Link
+	 */
+	tag: PropTypes.node,
```
Comment (jonnystening) [inline components/breadcrumb/src/Crumb.js:89]: @dominikwilkowski `PropTypes.node` give me a `checkPropTypes.js:20 Warning: Failed prop type: Invalid prop `tag` supplied to `Crumb`, expected a ReactNode.` 🤔

Comment (dominikwilkowski) [reply components/breadcrumb/src/Crumb.js:89]: Yeah cause your defaultProp is a string.... You should make that a component:
```
const Anchor = (props) => <a {...props} />
```

Comment (dominikwilkowski) [reply components/breadcrumb/src/Crumb.js:89]: To be clear: you shouldn't use defaultProps here but check if the props is set and then override the anchor. Using a defaultProp here as a string has to fail as we don't want people to just add strings.

Comment (jonnystening) [reply components/breadcrumb/src/Crumb.js:89]: First on my list was to ask you how to set a default setting a component as you had mentioned

Will set up in code, please hold, will commit shortly

Comment (jonnystening) [reply components/breadcrumb/src/Crumb.js:89]: Cos I can’t make the value a component (function) and PropTypes.node can be anything... incl string

**Pull request merged**